src/components/Tabs/Tabs.tsx

```diff
@@ -60,14 +49,23 @@ export default class Tabs extends React.PureComponent<Props, State> {
       containerWidth,
     );
 
-    this.setState({
+    return {
       visibleTabs,
       hiddenTabs,
-      tabToFocus: selected === nextProps.selected ? -1 : tabToFocus,
```

Comment:
Discussed this with Devon and we think it'll be best if we maintain the focus outline when selecting a tab from the popover